app/src/main/java/nl/hogeschoolrotterdam/projectb/fragment/MemoriesFragment.java

```diff
@@ -44,7 +55,25 @@ public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup c
         setHasOptionsMenu(true);
 
         memories = Database.getInstance().getMemories();
-        adapter = new MemoriesAdapter(memories);
+        actionModeCallback = new ActionModeCallback();
+        adapter = new MemoriesAdapter(items, new MemoriesAdapter.OnClickListener() {
+            @Override
+            public void onItemClick(View view, Memory obj, int pos) {
+                if (adapter.getSelectedItemCount() > 0) {
+                    enableActionMode(pos);
+                } else {
+                    Intent intent = new Intent(getContext(), MemoryDetailActivity.class);
+                    intent.putExtra("EXTRA_SESSION_ID", obj.getId());
+                    view.getContext().startActivity(intent);
+                    AnalyticsUtil.selectContent(getContext(), "SearchOrList");
```

Comment:
Please replace `"SearchOrList"` with `"List"`

It was always in the adapter, so we couldn't separate it at first. Now we can.